### ISSUE TITLE
Fixing Scrollbars not hiding them

### ISSUE THREAD

**Issue by bhavyaw** [MohammadYounes/AlertifyJS#114]
Hi,

First of all thanks for a great api. Super useful. 

I just have one query regarding the usage :

**Is there a way ( or a default property ) by which i can fix the scroll bars and not hide them. Hiding scrollbars makes the whole document jump from its position.**

When we show the alerts `.ajs-no-overflow` class is added to the body element. We can modify the css rule for this class as follows so that the scrollbars gets fixed.

`.ajs-no-overflow {
    outline: none;
    position: fixed;
    overflow: scroll;
    height: 100%;
    width: 100%;
}`

This can fix the scrollbar but whenever we'll add this class to the body document `position:fixed` property makes the document jump to the top. 

Long story short, is there some property which does this i.e makes scrollbars fixed.

**Btw this is what means by fixing scroll bars**

![fixed_scroll_bars](https://cloud.githubusercontent.com/assets/6906721/16953108/73c7d2c8-4de9-11e6-9c6a-598f5eaa2481.png)

**Comment by MohammadYounes** [comment]
Hi,

This is a duplicate of #112, see my https://github.com/MohammadYounes/AlertifyJS/issues/112#issuecomment-232935643.

Thanks

**Comment by MohammadYounes** [comment]
Actually, this solution is better than padding the body element.

The padding solution would not prevent shifting of fixed elements, and will introduce a visible gap on colored backgrounds.

![alertify](https://cloud.githubusercontent.com/assets/4712046/17207487/867258f0-54bd-11e6-93ac-3037d16581eb.PNG)

I'll implement this soon.

Thanks!

**Comment by MohammadYounes** [comment]
@bhavyaw  As of 1.8.0 you can use `alertify.defaults.preventBodyShift = true` to fix the scrollbar when showing a modal dialog. 

Thanks!
